roles/nova-data/tasks/libvirt.yml

```diff
@@ -56,29 +56,72 @@
   notify: restart nova services
   when: openstack_install_method == 'source'
 
+## Dev task, remove me before PR
 - name: update various lines in libvirtd.conf
-  lineinfile: dest=/etc/libvirt/libvirtd.conf regexp="{{ item.value.regexp }}"
-              line="{{ item.value.line }}"
+  lineinfile:
+    dest: /etc/libvirt/libvirtd.conf
+    regexp: "{{ item.value.regexp }}"
+    line: "{{ item.value.line }}"
+    state: absent
   with_dict:
-    ltcp:
+    listen_tcp:
       regexp: '^listen_tcp\s*='
       line: "listen_tcp = 1"
-    ltls:
+    listen_tls:
       regexp: '^listen_tls\s*='
-      line: "listen_tls = 0"
-    atcp:
+      line: "listen_tls = 1"
+    auth_tcp:
       regexp: '^auth_tcp\s*='
       line: 'auth_tcp = \"none\"'
+    auth_tls:
+      regexp: '^auth_tls\s*='
+      line: 'auth_tls = \"none\"'
+    no_verify_cert:
+      regexp: '^tls_no_verify_certificate\s*='
+      line: 'tls_no_verify_certificate = 1'
+    no_verify_ip:
+      regexp: '^tls_no_verify_address\s*='
+      line: 'tls_no_verify_address = 1'
+  notify: restart libvirt-bin
```

Comment:
I think this task can go away, we won't be listening. In fact, I wonder if we can turn of tcp listening all together.